pages/in/settings/others.vue

```diff
@@ -14,14 +18,66 @@
                     class="md-likecoin"
                   />
                   <div class="description">
-                    {{ $t('Register.form.enableEmail') }}
+                    {{ $t('Settings.others.email.enableEmail') }}
                   </div>
                 </li>
               </ul>
+            </div>
+          </div>
+
+          <div class="lc-container-3 lc-bg-gray-1 lc-padding-vertical-32">
+            <div class="flex">
+              <h2 class="lc-font-size-14 lc-font-weight-400">
+                {{ $t('Settings.others.likepay.title') }}
+              </h2>
+              <md-icon class="no-margin">
+                <a
+                  href="https://docs.like.co/developer/like-pay/web-widget/reference"
+                  target="_blank"
+                >
+                  help
+                </a>
+              </md-icon>
```

Comment:
```suggestion
              <md-button
                class="md-icon-button md-likecoin"
                href="https://docs.like.co/developer/like-pay/web-widget/reference"
                target="_blank"
              >
                <md-icon>help</md-icon>
              </md-button>
```

Reply:
The `<a>` is not preferable because of the native coding style of Vue or some other reasons?

Reply:
When we are using a UI framework, better be consistent and use a same component to implement a same feature. That way future modifications are easier.